pycqed/tests/test_flux_lutman.py

```diff
@@ -508,9 +512,18 @@ def test_partner_lutman_loading(self):
         self.fluxlutman.sq_amp(.3)
         self.fluxlutman_partner.sq_amp(.5)
         self.k0.reset_kernels()
-        self.fluxlutman.load_waveform_realtime(waveform_name='square')
-        np.testing.assert_allclose(self.AWG._realtime_w0[0], [.3])
-        np.testing.assert_allclose(self.AWG._realtime_w1[0], [.5])
+        self.fluxlutman.load_waveform_onto_AWG_lookuptable(waveform_name='square', regenerate_waveforms=True)
+        self.fluxlutman_partner.load_waveform_onto_AWG_lookuptable(waveform_name='square', regenerate_waveforms=True)
```

Comment:
The partner lutman should no longer be needed. The instrument driver will take care of updating the combined waveform and/or recompiling the program in case the waveform length has changed.